### DISCUSSION TITLE
Conditional augmentation and command tooling

### DISCUSSION THREAD

**Discussion by bradbeck** [cue-lang/cue#1688] [Q&A] [answered]
I'd like to understand if there is something that I'm doing wrong in the example at the following repository: https://github.com/bradbeck/cue-cmd-repro

The general idea is to define an `#Example` at the `policy` level and then be able to conditionally augment it based on the value of `#Example.id`.

```CUE
package repro

#Example: {
    id: string
}
```

The idea was to split the definition for each specific case into a separate file.

```CUE
package repro

#Example: {
    id: string
    if id == "one" {
        foo: bar: "baz"
    }
}
```

I use `#Example` at the `data` level to provide concrete values.

```CUE
package repro

#Example & {
    id: "one"
}
```

In order to get the evaluation result I'm looking for, I need to provide both the package and the path when doing the command invocation. e.g.

```bash
$ cue cmd print :repro ./policy/data
{"bar":"baz"}
```

```bash
$ cd policy/data
$ cue cmd print :repro .
{"bar":"baz"}
```

I feel I might be exercising a quirk in the system here.

**Comment by myitcv** [comment, marked as answer]
You've actually run into a bug: https://github.com/cue-lang/cue/issues/1700.

Thanks for starting a discussion in the first place to validate whether what you're seeing is expected or not.
